build/stf-run-ci/tasks/create_catalog.yml

```diff
@@ -76,7 +76,7 @@
           dockerfile: |
             # The base image is expected to contain
             # /bin/opm (with a serve subcommand) and /bin/grpc_health_probe
-            FROM registry.redhat.io/openshift4/ose-operator-registry:v4.12
+            FROM registry.redhat.io/openshift4/ose-operator-registry:v4.13
```

Comment:
👍🏻

Reply:
I'm actually :crossed_fingers: that this change solves the SCC problems, testing now with a local build using `__deploy_from_index_enabled` which I previously noted was broken.

Reply:
Still broken :-(